oak/server/rust/oak_loader/Cargo.toml

```diff
@@ -13,9 +13,13 @@ oak_debug = []
 default = ["oak_debug"]
 
 [dependencies]
+anyhow = "*"
```

Comment:
I think this should also be added to `oak/server/rust/oak_loader/BUILD`

Reply:
Done.